src/app/containers/MediaPlayer/index.jsx

```diff
@@ -92,9 +90,12 @@ const AudioVideoContainer = ({ blocks }) => {
       ) : null}
       <ParentWrapper>
         <ChildWrapper gridColumnStart={1} gridSpan={wrapperSpan}>
-          {platform === 'canonical' ? (
-            <Canonical id={pid} blocks={blocks} portrait={portrait} />
-          ) : null}
+          <ul>
+            <li>PID: {pid}</li>
+            <li>Orientation: {orientation}</li>
+            <li>Platform: {platform}</li>
+            <li>Portrait? {String(portrait)}</li>
+          </ul>
```

Comment:
I like this idea - however I think now we've removed the dial condition from `ArticleMain` this will always get output regardless of toggle state. So if it's non-trivial to make it conditional on the toggle I think we'll have to remove it/comment it out.

Reply:
Ah yes. I had not considered that. Will look into how feasible it is to include the toggle logic in this PR.

Reply:
Thanks - if it's a headache let's make a separate issue for it rather than let this PR get out of hand.

Reply:
Toggle support has been added.